test/ToucanRegenBridge.js

```diff
@@ -249,7 +251,9 @@ describe("Bridge contract", function () {
 
 		it("should fail with non-bridge admin account", async function () {
 			const tx = bridge.connect(broker).issueTCO2Tokens(regenUser, broker.address, tco2.address, 100, "test");
-			await expect(tx).to.be.revertedWith("invalid caller");
+			await expect(tx).to.be.revertedWith(
+				`AccessControl: account ${broker.address.toLowerCase()} is missing role ${TOKEN_ISSUER_ROLE}`
```

Comment:
This would not be a helpful error message because it displays a hash.

```suggestion
				`AccessControl: account ${broker.address.toLowerCase()} is missing TOKEN_ISSUER_ROLE`
```